common/persistence/serialization/persistence_mapper_test.go

```diff
@@ -54,8 +55,8 @@ func TestInternalWorkflowExecutionInfo(t *testing.T) {
 		LastFirstEventID:                   int64(rand.Intn(1000)),
 		LastEventTaskID:                    int64(rand.Intn(1000)),
 		LastProcessedEvent:                 int64(rand.Intn(1000)),
-		StartTimestamp:                     time.Now(),
-		LastUpdatedTimestamp:               time.Now(),
+		StartTimestamp:                     time.UnixMilli(1752018142820),
```

Comment:
These tests compare the structs field by field because the round-trip of the mapper strips some of the precision off of the time (we don't store nanosecond accuracy).

If we instead use a fixed millisecond value then we can compare the entire structs.